content/sensu-go/5.15/guides/use-federation.md

```diff
@@ -0,0 +1,313 @@
+
+{{< language-toggle >}}
+
+{{< highlight yml >}}
+---
+api_version: federation/v1
+type: EtcdReplicator
+metadata:
+  name: AlphaClusterRoleBindings
+spec:
+  ca_cert: "/etc/sensu/certs/ca.pem"
+  cert: "/etc/sensu/certs/cert.pem"
+  key: "/etc/sensu/certs/key.pem"
+  url: https://sensu.alpha.example.com:2379
+  api_version: core/v2
+  resource: ClusterRoleBinding
+  replication_interval_seconds: 30
+{{< /highlight >}}
+
+{{< highlight json >}}
+{
+  "api_version": "federation/v1",
+  "type": "EtcdReplicator",
+  "metadata": {
+    "name": "AlphaClusterRoleBindings"
+  },
+  "spec": {
+    "ca_cert": "/etc/sensu/certs/ca.pem",
+    "cert": "/etc/sensu/certs/cert.pem",
+    "key": "/etc/sensu/certs/key.pem",
+    "url": "https://sensu.alpha.example.com:2379",
+    "api_version": "core/v2",
+    "resource": "ClusterRoleBinding",
+    "replication_interval_seconds": 30
+  }
+}
+{{< /highlight >}}
+
+{{< /language-toggle >}}
+
+{{< language-toggle >}}
+
+{{< highlight yml >}}
+---
+api_version: federation/v1
+type: EtcdReplicator
+metadata:
+  name: BetaClusterRoleBindings
+spec:
+  ca_cert: "/etc/sensu/certs/ca.pem"
+  cert: "/etc/sensu/certs/cert.pem"
+  key: "/etc/sensu/certs/key.pem"
+  url: https://sensu.beta.example.com:2379
+  api_version: core/v2
+  resource: ClusterRoleBinding
+  replication_interval_seconds: 30
+{{< /highlight >}}
+
+{{< highlight json >}}
+{
+  "api_version": "federation/v1",
+  "type": "EtcdReplicator",
+  "metadata": {
+    "name": "BetaClusterRoleBindings"
+  },
+  "spec": {
+    "ca_cert": "/etc/sensu/certs/ca.pem",
+    "cert": "/etc/sensu/certs/cert.pem",
+    "key": "/etc/sensu/certs/key.pem",
+    "url": "https://sensu.beta.example.com:2379",
+    "api_version": "core/v2",
+    "resource": "ClusterRoleBinding",
+    "replication_interval_seconds": 30
+  }
+}
+{{< /highlight >}}
+
+{{< /language-toggle >}}
+
+To configure this EtcdReplicator on our `gateway` cluster, first use `sensuctl config view` to verify that `sensuctl` is configured to talk to a `gateway` cluster API. Reconfigure `sensuctl` as necessary.
+
+With the above EtcdReplicator definition written to disk as `AllClusterRoleBindings.yml`, use `cat AllClusterRoleBindings.yml | sensuctl create`  to create the EtdReplicator on the `gateway` cluster.
+
+Our [etcd-replicators reference][2] includes [examples][9] for `Role`, `RoleBinding`, `ClusterRole`, and `ClusterRoleBinding` resources. For a consistent experience, repeat the above example for `Role`, `RoleBinding` and `ClusterRole` resource types.
+
+You can verify that the EtcdReplicator resource is working as expected by reconfiguring `sensuctl` to communicate with the `alpha` and `beta` clusters and issuing the `sensuctl cluster-role-binding list` command. You should see 
```

Comment:
```suggestion
To verify that the EtcdReplicator resource is working as expected, reconfigure `sensuctl` to communicate with the `alpha` and `beta` clusters and issue the `sensuctl cluster-role-binding list` command. You should see 
```

Reply:
I think there's an incomplete thought here at the end -- "You should see..."

Reply:
```suggestion
To verify that the EtcdReplicator resource is working as expected, reconfigure `sensuctl` to communicate with the `alpha` and then `beta` clusters, issuing the `sensuctl cluster-role-binding list` command for each. You should see the `federation-viewer-readonly` binding created in step 3 listed in the output from each cluster:

{{< highlight shell >}}
$ sensuctl cluster-role-binding info federation-viewer-readonly
=== federation-viewer-readonly
Name:         federation-viewer-readonly
Cluster Role: view
Subjects:     
  Users:      federation-viewer
{{< /highlight >}}
```